src/lib/rlang.h

```diff
@@ -85,8 +87,15 @@ static inline sexp* KEEP_N(sexp* x, int* n) {
 #include "vec-list.h"
 
 
-static inline void rlang_init_library() {
+// This *must* be called before making any calls to the functions
+// provided in the library
+static inline void r_init_library() {
   r_dot_environment_sym = r_sym(".Environment");
+
+  r_quo_get_expr = (sexp* (*)(sexp*)) r_peek_c_callable("rlang", "rlang_quo_get_expr");
+  r_quo_set_expr = (sexp* (*)(sexp*, sexp*)) r_peek_c_callable("rlang", "rlang_quo_set_expr");
+  r_quo_get_env = (sexp* (*)(sexp*)) r_peek_c_callable("rlang", "rlang_quo_get_env");
+  r_quo_set_env = (sexp* (*)(sexp*, sexp*)) r_peek_c_callable("rlang", "rlang_quo_set_env");
```

Comment:
I think it would be more readable to typedef some function types.

Reply:
my general approach with the library is to give a chance to C syntax. I.e. trying not to hide structs, enums, and pointers with typedefs. So far I am enjoying the explicitness it brings. In this  particular case I also think anonymous function pointers are readable as you can immediately see the number and types of arguments and return value.

Reply:
Right. I agree that this is clean now. But I suppose the api will grow. Will you ever have non-sexp arguments for example? Making a SEXP in C code is not always needed. 

Anyway, I agree that this is not an issue right now.